src/ui/default-prop-types.js

```diff
@@ -16,14 +17,17 @@ export const sectionEditorProps = {
   onContentChanged: PropTypes.func.isRequired
 };
 
+export const importSourceShape = PropTypes.shape({
+  name: PropTypes.string.isRequired,
+  hostName: PropTypes.string.isRequired,
+  allowUnsecure: PropTypes.bool
+});
```

Comment:
There is no need to inject these through initial state as it is already present in the client config.

Reply:
Sorry, that's already the case, please ignore the comment !